### PR TITLE
queue related features

### PR DESCRIPTION
**Pull request by shachopin**

### CODE REVIEW

File: spec/controllers/queue_items_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+  describe "POST create" do
+    it "redirect to the my queue page" do
+      session[:id] = Fabricate(:user).id
+      video = Fabricate(:video)
+      post :create, video_id: video.id
+      expect(response).to redirect_to my_queue_path
+    end
+
+    it "create a queue item" do
```
Comment (falonofthetower) [inline spec/controllers/queue_items_controller_spec.rb:28]: `it "creates a queue item" do`

File: spec/fabricators/queue_item_fabricator.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Fabricator(:queue_item) do
+
```
Comment (falonofthetower) [inline spec/fabricators/queue_item_fabricator.rb]: You don't need this line

Comment (shachopin) [inline spec/fabricators/queue_item_fabricator.rb]: which line? You mean just put something like Fabricator(:queue_item) do;end ?

Comment (falonofthetower) [inline spec/fabricators/queue_item_fabricator.rb]: I actually just meant line 2 is unnecessary white space. Personally I don't have a problem with leaving the method on two lines, but you definitely shouldn't have third blank line as that is rather universally bad.

Comment (shachopin) [inline spec/fabricators/queue_item_fabricator.rb]: got it . thanks!

**Pull request merged**